spec/js-api/compile.d.ts.md

```diff
@@ -91,6 +110,21 @@ export function compileAsync(
 
 Compiles the Sass `source`:
 
+* If any object in `options.importers` is exactly equal to the object
+  `nodePackageImporter`:
+
+  * If no filesystem is available, throw an error.
+
+    > This primarily refers to a browser environment, but applies to other
+    > sandboxed JavaScript environments as well.
+
+  * Let `pkgImporter` be a [Node Package Importer] with an associated
+    `entryPointURL` of `options.url` if it is a `file:` URL or the current
+    working directory otherwise.
+
+  * Replace `nodePackageImporter` with `pkgImporter` in a copy of
+    `options.importers`.
```

Comment:
I don't think this is accurate any more.

Reply:
Applied draft changes in ec40399d98f7c378125f207f5ec375e9f83e42ec